<launch>

    <!-- SENSORS -->
    <node name="left_encoder" pkg="crues_sensors" type="enc_node.py">
        <param name="pin_a" type="str" value="ela" />
        <param name="pin_b" type="str" value="elb" />
        <param name="rate" type="double" value="50" />
        <remap from="wheel" to="lwheel"/>
    </node>
    <node name="right_encoder" pkg="crues_sensors" type="enc_node.py">
        <param name="pin_a" type="str" value="era" />
        <param name="pin_b" type="str" value="erb" />
        <param name="rate" type="double" value="50" />
        <remap from="wheel" to="rwheel"/>
    </node>

    <!-- ACTUATORS -->
    <node name="motor_sleep" pkg="crues_actuators" type="motor_sleep_node.py" />
    <node name="left_motor" pkg="crues_actuators" type="drive_node.py">
        <param name="pwm_pin" type="str" value="mlp" />
        <param name="dir_pin" type="str" value="mld" />
        <param name="range_min" type="double" value="-0.5" />
        <param name="range_max" type="double" value="0.5" />
        <remap from="motor_cmd" to="lmotor_cmd" />
    </node>
    <node name="right_motor" pkg="crues_actuators" type="drive_node.py">
        <param name="pwm_pin" type="str" value="mrp" />
        <param name="dir_pin" type="str" value="mrd" />
        <param name="range_min" type="double" value="-0.5" />
        <param name="range_max" type="double" value="0.5" />
        <remap from="motor_cmd" to="rmotor_cmd" />
    </node>

    <!-- CONTROL -->
    <node name="joust_bot" pkg="crues_control" type="joust_bot_control.py" />

</launch>